modules/jaeger-config-storage.adoc

```diff
@@ -0,0 +1,207 @@
+////
+This REFERENCE module included in the following assemblies:
+-rhbjaeger-deploy.adoc
+////
+
+[id="ossm-jaeger-config-agent_{context}"]
+= Jaeger storage configuration options
+
+The query and collector services are configured with a supported storage type; Elasticsearch.  Multiple instances of each of these components can be provisioned as required for performance and resilience purposes.
+
+When there is no `es.server-urls` option as part of a Jaeger production instance and Elasticsearch is set as the storage type, the Jaeger Operator creates an Elasticsearch cluster via the Elasticsearch Operator by creating a Custom Resource based on the configuration provided in storage section. The Elasticsearch cluster is meant to be dedicated for a single Jaeger instance.
+
+[CAUTION]
+====
+There can be only one Jaeger with self-provisioned Elasticsearch instance per namespace.
+====
+
+.Jaeger storage parameters
+[options="header"]
+[cols="l, a, a, a"]
+|===
+|Parameter |Description |Values |Default value
+
+
+|spec:
+ storage:
+  options: {}
+|Configuration options that define the storage.
+|
+|
+
+|storage:
+  type:
+|Type of storage to use for the deployment.
+|`memory` or `elasticsearch`
+|`memory`
+
+|storage:
```

Comment:
As mentioned in the collector section, there are other ES options - the `es` and `es-archive` prefixed CLI flags from https://www.jaegertracing.io/docs/1.17/cli/#jaeger-collector-elasticsearch.